server/src/main/java/org/candlepin/model/Product.java

```diff
@@ -519,9 +776,16 @@ public void setDependentProductIds(Set<String> dependentProductIds) {
     @Override
     public String getHref() {
         // If we don't have an owner here, we're in a bit of trouble.
-        return (this.getOwner() != null && this.getOwner().getKey() != null && this.getId() != null) ?
-            "/owners/" + this.getOwner().getKey() + "/products/" + this.getId() :
-            "";
+        // PER-ORG PRODUCT VERSIONING TODO:
+        // Do we have a way to determine the current owner from context? We need a way to reference
+        // a specific product (uuid) but with owner permissions...? Should this just change to UUIDs
+        // everywhere? This is messy.
+
```

Comment:
The product and content UUIDs are to be completely internalized, as with the converging and diverging products, it's entirely possible that a RHID for a given org could point to one UUID one day, and another the next.